integration/deterministic-build-output-test.ts

```diff
@@ -15,31 +15,63 @@ test("builds deterministically under different paths", async () => {
 
   // Virtual modules tested:
   //  * browserRouteModulesPlugin (implicitly tested by root route)
+  //  * cssEntryModulePlugin (implicitly tested by build)
+  //  * cssModulesPlugin (via app/routes/foo.tsx' CSS Modules import)
   //  * emptyModulesPlugin (via app/routes/foo.tsx' server import)
   //  * mdx (via app/routes/index.mdx)
   //  * serverAssetsManifestPlugin (implicitly tested by build)
   //  * serverEntryModulePlugin (implicitly tested by build)
   //  * serverRouteModulesPlugin (implicitly tested by build)
   let init = {
     files: {
+      "remix.config.js": js`
+        module.exports = {
+          future: {
+            unstable_cssBundle: true
```

Comment:
I wonder if we could also make this config a bit more granular? That way we could continue experimenting with the same API in the future once CSS Modules are stable.

```ts
{
  future: {
    unstable_cssBundle: {
      cssModules: true, 
      vanillaExtract: true,
    },
  },
}
```

Reply:
I'm good with more granular but I'd rather not have nested flags, just keep them all flat.

Reply:
I've flattened it to `future.unstable_cssModules`.